public/javascripts/login.js

```diff
@@ -1,54 +1,44 @@
-$(document).ready(function() {
-
+window.onload = function() {
```

Comment:
that's the way firebase suggests, and i tried removing it but the other methods fail / throw errors